plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/WooCommercePayments.php

```diff
@@ -212,11 +219,33 @@ function ( $plugins ) {
 	 *
 	 * @return \WC_Payments|null
 	 */
-	private static function get_woo_payments_gateway() {
+	private static function get_gateway() {
 		$payment_gateways = WC()->payment_gateways->payment_gateways();
 		if ( isset( $payment_gateways['woocommerce_payments'] ) ) {
 			return $payment_gateways['woocommerce_payments'];
 		}
 		return null;
 	}
+
+	/**
+	 * Check if the store has any enabled ecommerce gateways, other than WooPayments.
+	 *
+	 * We exclude offline payment methods from this check.
+	 *
+	 * @return bool
+	 */
+	public static function has_other_ecommerce_gateways(): bool {
+		$gateways         = WC()->payment_gateways->get_available_payment_gateways();
+		$enabled_gateways = array_filter(
+			$gateways,
+			function ( $gateway ) {
+				// Filter out any WooPayments-related or offline gateways.
+				return 'yes' === $gateway->enabled
+					&& 0 !== strpos( $gateway->id, 'woocommerce_payments' )
+					&& ! in_array( $gateway->id, array( 'bacs', 'cheque', 'cod' ), true );
```

Comment:
You've likely already checked - is it possible to get the list of offline methods from an existing WooCommerce API?

Reply:
We don't have such an API.